### PR TITLE
v0.18.1

### PR DESCRIPTION
**Pull request by wash2**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,22 @@
 
 #### Additions
 
+## 0.18.1 - 2023-11-04
+
+#### Breaking Changes
+
```
Comment (kchibisov) [inline CHANGELOG.md:14]: I'd rather remove this section, since it's empty

**Comment by Friz64** [comment]
#430 is a breaking change, no? If so, could this still be released, but without #430?

**Comment by kchibisov** [comment]
Hm, indeed, the release must reside on its own branch and not on master. I haven't noticed it, but thx.

**Comment by wash2** [comment]
Sure ill make the change

**Comment by tmccombs** [comment]
Any progress on this? I need https://github.com/Smithay/client-toolkit/commit/35646632b93fe8b88cf8b9b095ea5166c9063a58#diff-67e767f21e99d1960557cb27ed6937bf34d8be4fdd30fb9fe6ae31f186e0f17c to be able to use `delegate_pointer!` with a custom pointer data type.

**Comment by wash2** [comment]
Ya, I will do a release for 0.18.1 soon, and I will close this PR afterward.

**Pull request closed without merging**